src/Microsoft.ML.OnnxTransform.StaticPipe/DnnImageFeaturizerStaticExtensions.cs

```diff
@@ -8,9 +8,8 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.StaticPipe;
 using Microsoft.ML.StaticPipe.Runtime;
-using Microsoft.ML.Transforms;
 
-namespace Microsoft.ML.OnnxTransform.StaticPipe
+namespace Microsoft.ML.Transforms.StaticPipe
```

Comment:
>Transforms [](start = 23, length = 10)

Reason why I make this change is because I don't like what I have to do this:
`new Transforms.OnnxTransform(...);`
instead of just 
` new OnnxTransform(...);`
if we have Microsoft.ML in usings. (see OnnxTransformTests.cs) 

Shall we just call it Microsoft.ML.StaticPipe without any additional sub namespaces?

Reply:
I'd expect that static pipe would be something we'd nail down later, but I'd *expect* that inputs to to staticpipe would just be accomplished by taking a `PiplineColumn` instance, and applying the transform to it. There should not be any `new Transforms.OnnxTransform` in the static pipeline at all, since the static pipeline is more declarative. So I don't like either of the things you've described, if indeed that is now it iss set up. If that's not how it was set up, well, that's bad. :) So that would be something to file and issue on and fix. (Though I think static pipeline is something we're going to refine more post v1.0.)

---
In reply to: [245456891](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245456891) [](ancestors = 245456891)

Reply:
Actually, from what I see below, it *is* set up as an extension method on the columns. So what is the problem?

---
In reply to: [245458003](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245458003) [](ancestors = 245458003,245456891)

Reply:
If you have your code and you use `using Microsoft.ML;` in it (which we do all the time), I can't longer just create OnnxTransform object via `new OnnxTransform(...)`, because you will get collision of namespace and object.
So you have to directly specify what your transform is in Transform namespace like this: `Transforms.OnnxTransform` which I found annoying.

---
In reply to: [245458074](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245458074) [](ancestors = 245458074,245458003,245456891)

Reply:
Oh I see. Yes I agree then, I didn't know you were talking about namespace/object collision.

Ultimately all static extensions should be in `Microsoft.ML` namespace (since we must accept that inclusion of a specific nuget is license to use it, similar to our attitude about `MLContext` extensions), but not right now.

---
In reply to: [245458427](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245458427) [](ancestors = 245458427,245458074,245458003,245456891)

Reply:
But that happens only when ppl use both static and dynamic, right? Our tests do so, but idk how many ppl will mix ant match static and dynamic. 

---
In reply to: [245458713](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245458713) [](ancestors = 245458713,245458427,245458074,245458003,245456891)